core/src/main/kotlin/xtdb/indexer/LiveTable.kt

```diff
@@ -26,6 +28,16 @@ class LiveTable
 ) : AutoCloseable {
 
     private val trieWriter = TrieWriter(al, bp, false)
+    private val trieMetadataBuilder = TrieMetadata.newBuilder()
+        .setMinValidFrom(MAX_LONG)
```

Comment:
may be able to default these in the protobuf instead...? :thinking:

Reply:
I put it in the proto for now. The important bit to keep in mind is that if you build a TrieMetadata without calculating anything on it, the contained data doesn't make much sense.